docs/README.md

```diff
@@ -121,5 +127,421 @@ something like
 
 ```shell
 $ boost version
-@boostercloud/cli/0.3.3 darwin-x64 node-v13.12.0
+@boostercloud/cli/0.4.1 darwin-x64 node-v13.12.0
+```
+
+### Your first Booster app in 10 minutes
+
+In this section, we will go through all the necessary steps to have the backend up and
+running for a blog application in just a few minutes. The steps to follow will be:
+
+- [Create project](#1-create-the-project)
+- [First command](#2-first-command)
+- [First event](#3-first-event)
+- [First entity](#4-first-entity)
+- [First read model](#5-first-read-model)
+- [Deployment](#6-deployment)
+- [Testing](#7-testing)
+  - [Creating posts](#71-creating-posts)
+  - [Retrieving all posts](#72-retrieving-all-posts)
+  - [Retrieving specific post](#73-retrieving-specific-post)
+- [Removing stack](#8-removing-stack)
+- [More functionalities](#9-more-functionalities)
+
+#### 1. Create the project
+
+First of all, we will use the Booster CLI to create a project. Run this command and follow
+the instructions, when asked for the provider, select AWS as that is what we have
+configured [here](#set-up-an-aws-account)
+
+```shell
+> boost new:project boosted-blog
+
+...
+
+ℹ boost new 🚧
+✔ Creating project root
+✔ Generating config files
+✔ Installing dependencies
+ℹ Project generated!
+```
+
+> Booster's new: commands follows this structure
+>
+> - `boost` is the Booster CLI
+> - `new:<resource>` new is a CLI command, :project tells booster the kind of resource
+> - `boosted-blog` is a parameter for the `new:project` command
+> project name
+
+The `new:project` command generates some scaffolding for you. The project name will be the
+project's root so `cd` to it:
+
+```shell
+cd boosted-blog
+```
+
+There you should have these files and directories already generated:
+
+```text
+boosted-blog
+├── .eslintignore
+├── .gitignore
+├── package-lock.json
+├── package.json
+├── src
+│   ├── commands
+│   ├── common
+│   ├── config
+│   │   └── config.ts
+│   ├── entities
+│   ├── events
+│   └── index.ts
+└── tsconfig.json
+```
+
+Now open the project in your favorite editor, e.g. [Visual Studio Code](https://code.visualstudio.com/).
+
+#### 2. First command
+
+Commands define the input to our system, so we'll start by generating our first [command](#commands) to create posts. Use the command generator in the project's root directory as follows:
+
+```bash
+boost new:command CreatePost --fields postId:UUID title:string content:string author:string
+```
+
+The `new:command` generator creates a `CreatePost.ts` file in the `commands` folder:
+
+```text
+boosted-blog
+└── src
+    └── commands
+        └── CreatePost.ts
+```
+
+We still need to define two more things:
+
+1. Who is authorized to run this command.
+1. And what events are being triggered when the command is executed.
```

Comment:
```suggestion
2. And what events are being triggered when the command is executed.
```

Reply:
For MD it doesn't matter what number you use because the parser will handle the ordered list automatically as soon as they start with `<number>.`

https://www.markdownguide.org/basic-syntax#ordered-lists

I think it is actually a good practice to just use `1.` because if you move the item you don't have to care about the new order